cogs/welcome/welcome.py

```diff
@@ -1,13 +1,16 @@
 """Welcome cog
 Sends welcome DMs to users that join the server.
 """
-
 import asyncio
 import discord
 import logging
 import random
+import aiohttp
+import os
+import io
 
-from redbot.core import Config, checks, commands
+from PIL import Image, ImageChops, ImageOps
+from redbot.core import Config, checks, commands, data_manager
```

Comment:
Please sort these imports.

```suggestion
import asyncio
import io
import logging
import os
import random

import aiohttp
import discord
from PIL import Image, ImageChops, ImageOps

from redbot.core import Config, checks, commands, data_manager
```